v21.2/cluster-setup-troubleshooting.md

```diff
@@ -362,7 +370,7 @@ Symptoms of disk stalls include:
 
 Causes of disk stalls include:
 
-- Disk operations have backed up due to underprovisioned IOPS. Make sure you are deploying with our [recommended production settings for storage](recommended-production-settings.html#storage).
+- Disk operations have backed up due to underprovisioned IOPS. Make sure you are deploying with our [recommended production settings for storage](recommended-production-settings.html#storage) and [monitoring disk IOPS](common-issues-to-monitor.html#disk-iops).
```

Comment:
This is a super nit (and it might be because I always have backups on the brain), but here "backed up" is meaning slowed or in some kind of queue? I wondered if there was possibility for confusion there — in terminology. But again, might just be me!